### ISSUE TITLE
Better Expose Auto Login Feature

### ISSUE THREAD

**Issue by anthonylavado** [jellyfin/jellyfin-androidtv#223]
As per some issues on GitHub, it's not clear that there is an auto-login feature in the settings.

Right now, to access this (assuming you have launched the app and logged in):
- Scroll down to Settings
- Select "Startup Behaviour"
- Change it from "Show login screen" to "Automatically login as this user"

Perhaps this needs to be explained clearer? Or maybe add a "remember me/auto login" checkbox when signing in?

**Comment by MrChip53** [comment]
@thornbill @nielsvanvelzen Any planned implementations to improve this?

**Comment by thornbill** [comment]
Yeah. I'm currently working on rewriting everything related to how the app handles authentication.

**Comment by MrChip53** [comment]
So I noticed the login flow of the release app is significantly different than the master branch. Is this because you all scrapped the old login and havent implemented a new login or do you just sort of "back port" changes to the release branch? Also, are you or anyone else working on multi-user support?

**Comment by nielsvanvelzen** [comment]
When I updated the app to use the current apiclient version I didn't bother making changes to the authentication flow because it would take quite some time and @thornbill is already working on that.

That's the reason I implemented only the alerts and it's an even worse experience as before (who knew that was possible?!)

**Comment by MrChip53** [comment]
Ok. No problem. Im just trying to get an idea of what issues I could maybe work on but dont want to do something someone else already is.

As far as the login system goes, I was thinking it would be nice if it auto saves all users that you log in as and on startup pulls up a user selection screen like... netflix. You can have it save server credentials seperately like before or not. I think if there is an "Add User" button popping up the alerts to login is sufficient while maybe still less than ideal(so maybe save server credentials to pick existing server or add new, the log in to user?).

My understanding is also the server devs want the clients to handle 100% of the multi user support.

I don't know what thornbill has got worked up though.

**Comment by thornbill** [comment]
I haven't worked through all the implementation details yet, but supporting multiple saved logins is definitely planned.

Really all alerts need to be removed as they do not work properly with remotes (see: #167) and Google recommends to use the guided activity style interface for alerts.

**Comment by nielsvanvelzen** [comment]
Closing this as this issue is fixed now. For more information on the login changes see #612
